components/stlViewer/index.tsx

```diff
@@ -55,23 +55,28 @@ export default function StlViewer({
 					e.preventDefault();
 					orbitControls.enableZoom = false;
 
-					const pos = camera.position
+					const pos = camera.position;
 					let x = 0;
 					let y = 0;
 					let z = 0;
 					if (pos.x > pos.y && pos.x > pos.z) {
-						x = 0.1;
-						y = pos.y / (pos.x * 10)
-						z = pos.z / (pos.x * 10)
+						x = 0;
+						y = 0.2;
+						z = 0;
+						console.log("xxxxxxxxxxxx");						
 					} else if (pos.y > pos.x && pos.y > pos.z) {
-						y = 0.1;
-						x = pos.y / (pos.y * 10)
-						z = pos.z / (pos.y * 10)
+						y = 0.2;
+						x = 0;
+						z = 0;
+						console.log("yyyyyyyyyyyy")
```

Comment:
remove this console log

Reply:
each conditional statement determines the direction of the rotation based on camera view.